ts/src/base/functions/customThrottle.ts

```diff
@@ -64,15 +63,15 @@ class Throttler {
                 if (highestPriorityQueue.length === 0) {
                     delete this.priorityQueue[highestPriorityIndex];
                 }
-                if (this.queue.length === 0 && Object.keys (this.priorityQueue).length === 0) {
+                if (Object.keys (this.priorityQueue).length === 0) {
```

Comment:
If all 3 priority queues got empty, you'd have something like this: 

```
{
1: [],
2: [],
3: []
}
```

Where do you delete all keys to make sure `Object.keys (this.priorityQueue).length === 0` retuns false in this case?